apps/frontend/src/app/layout.tsx

```diff
@@ -17,7 +21,19 @@ export default function RootLayout({
 }>) {
   return (
     <html lang='hu'>
-      <body className={inter.className}>{children}</body>
+      <body className={`${GeistSans.className}`}>
+        <ThemeProvider attribute='class' defaultTheme='dark' enableSystem disableTransitionOnChange>
```

Comment:
We should probably also have a switcher for it. So people can use the other theme, that is not their system default.

Reply:
It was kinda intentional leaving it out, so we get a consistent dope look, just like spotify does with its dark theme. But I guess I can make a switch and later if we decide we won't need it we can remove it anytime.

Reply:
But then what is the point of a theme provider? We can just make everything dark. Also for me it appears in white mode for some reason.
![image](https://github.com/user-attachments/assets/09ce3d49-bd3c-475e-ab20-011372155c98)

Reply:
Interesting..